shub/settings.py

```diff
@@ -0,0 +1,669 @@
+    # GitLab OAuth2
+    "SOCIAL_AUTH_GITLAB_KEY": None,
+    "SOCIAL_AUTH_GITLAB_SECRET": None,
+    # Google Cloud Build + Storage: configure a custom builder and storage endpoint
+    "GOOGLE_APPLICATION_CREDENTIALS": None,  # /path/to/credentials.json
+    "GOOGLE_PROJECT": None,
+    # After build, do not delete intermediate dependencies in cloudbuild bucket (keep them as cache for rebuild if needed).
+    # Defaults to being unset, meaning that files are cleaned up. If you define this as anything, the build files will be cached.
+    "GOOGLE_BUILD_CACHE": None,  # "true"
+    # if you want to specify a version of Singularity. The version must coincide with a container tag hosted under singularityware/singularity.
+    # The version will default to 3.2.0-slim If you want to use a different version, update this variable.
+    "GOOGLE_BUILD_SINGULARITY_VERSION": None,  # "v3.2.1-slim"
+    # is the name for the bucket you want to create. The example here is using the unique identifier appended with “sregistry-"
+    # If you don't define it, it will default to a string that includes the hostname.
+    # Additionally, a temporary bucket is created with the same name ending in _cloudbuild. This bucket is for build time dependencies,
+    # and is cleaned up after the fact. If you are having trouble getting a bucket it is likely because the name is taken,
+    # and we recommend creating both <name> and <name>_cloudbuild in the console and then setting the name here.
+    "GOOGLE_STORAGE_BUCKET": None,  # taco-singularity-registry"
+    # Bitbucket social auth
+    "SOCIAL_AUTH_BITBUCKET_OAUTH2_KEY": None,  # '<your-consumer-key>'
+    "SOCIAL_AUTH_BITBUCKET_OAUTH2_SECRET": None,  # '<your-consumer-secret>'
+    # LDAP Authentication (ldap-auth)
+    # Only required if 'ldap-auth' is added to PLUGINS_ENABLED in config.py
+    # This example assumes you are using an OpenLDAP directory
+    # If using an alternative directory - e.g. Microsoft AD, 389 you
+    # will need to modify attribute names/mappings accordingly
+    # See https://django-auth-ldap.readthedocs.io/en/1.2.x/index.html
+    # The URI to our LDAP server (may be ldap:// or ldaps://)
+    "AUTH_LDAP_SERVER_URI": None,  # "ldaps://ldap.example.com
+    # DN and password needed to bind to LDAP to retrieve user information
+    # Can leave blank if anonymous binding is sufficient
+    "AUTH_LDAP_BIND_DN": "",
+    "AUTH_LDAP_BIND_PASSWORD": "",
+    "AUTH_LDAP_USER_SEARCH": None,  # "ou=users,dc=example,dc=com"
+    "AUTH_LDAP_GROUP_SEARCH": None,  # "ou=groups,dc=example,dc=com"
+    # Anyone in this group can get a token to manage images, not superuser
+    "AUTH_LDAP_STAFF_GROUP_FLAGS": None,  # "cn=staff,ou=django,ou=groups,dc=example,dc=com",
+    # Anyone in this group is a superuser for the app
+    "AUTH_LDAP_SUPERUSER_GROUP_FLAGS": None,  # "cn=superuser,ou=django,ou=groups,dc=example,dc=com"
+    # "cn=sregistry_admin,ou=groups,dc=example,dc=com"
```

Comment:
Not sure what this LDAP string is for -- is it a second example for `AUTH_LDAP_SUPERUSER_GROUP_FLAGS` above?

Reply:
yes, just a second example! I had two previously and wanted to keep both of them for the developer user. I'm not great with LDAP so I couldn't tell you the difference, I'd have to ping the plugin contributor to get details.

Reply:
I would remove this line then as I don't think it provides any additional information for someone who is integrating with LDAP -- the other one is sufficient. (Says someone who worked on LDAP code for 10 years.)